node/src/types/block.rs

```diff
@@ -1180,7 +1473,7 @@ impl Block {
     /// Generates a random instance using a `TestRng`.
     #[cfg(test)]
     pub fn random(rng: &mut TestRng) -> Self {
-        let era = rng.gen_range(0..5);
+        let era = rng.gen_range(1..6);
```

Comment:
I still think we should find a way to allow `0` here again.

Reply:
There's a ticket for this: https://github.com/casper-network/casper-node/issues/1789